features/graph/common/src/main/java/org/opennms/oce/features/graph/common/GraphMLConverter.java

```diff
@@ -191,27 +194,28 @@ private void handleEdge(Edge e) {
         e.getInventoryObjectPeerRef().ifPresent(peerRef ->  edge.setProperty(ONMS_GRAPHML_LABEL, "peer reference"));
         e.getInventoryObjectRelativeRef().ifPresent(relativeRef ->  edge.setProperty(ONMS_GRAPHML_LABEL, "relative reference"));
 
-        Collection<Vertex> incidentVertices = g.getIncidentVertices(e);
-        if (incidentVertices.size() == 1) {
-            Vertex v = incidentVertices.iterator().next();
+        Collection<Vertex> situationVertices = g.getIncidentVertices(e);
```

Comment:
I don't think this should change. Incident vertices is the graph term for the vertices connected to the edge, it isn't related to our term of incidents/situations. That applies below as well.

Pretty annoying that term was overloaded!